### ISSUE TITLE
Why???

### ISSUE THREAD

**Issue by Botspot** [SkywalkerSW5/Rpi686#4]
I think this project has great potential. Especially if you can get Virtualbox running.

Why are you discontinuing both rpi686 and PiRoot? Are you working to release an improved x86 linux chroot under yet another name?

**Comment by SkywalkerSW5** [comment]
The problem is that in the end, piRoot and Rpi686 are just wrappers for QEMU. All they do is launch/generate a x86 Linux VM. That means any of the emulation problems were with QEMU. There were many issues, and the main one was something involving a segmentation fault. This prevented many apps from running at all. And there was nothing I could do. They were just some shell scripts and a base system. If you wanted to, you could generate one yourself. 

And I doubt I would make another x86 emulator for the Raspberry Pi because I don't use it anymore as my primary desktop. The reason I made most of the RPI related projects was because I needed them for day to day usage. However, I now have a "real" computer so, I stopped those projects.

**Comment by Botspot** [comment]
Interesting. I got tired of WINE's segfaults, and so gave up.  
But then I discovered [box86](https://github.com/ptitSeb/box86), which is way more stable, and way faster too.

Perhaps that would help? Others say it runs chroots really well.

**Comment by SkywalkerSW5** [comment]
I haven't looked too deep into it, but from light looking, piRoot is a super trimmed down version of it. I honestly encourage anyone who still uses piRoot to switch over to that, because I highly doubt I am gonna do anything for the Raspberry Pi specifically. 

Another reason though for piRoot's end was lack of a user base. I doubt much people were using it, and when I started ending it, no forks were made so I assumed nobody used it.

PS - I think I did get Virtualbox running at some point with piRoot, but it required some tinkering. If i find any thing related to it I will post it here :}
